lib/GuestManager.php

```diff
@@ -228,25 +230,39 @@ public function sendEmailInvitation(Room $room, Participant $participant): void
 		]);
 
 		if ($user instanceof IUser) {
-			$subject = $this->l->t('%s invited you to a conversation.', $user->getDisplayName());
+			$subject = $this->l->t('%1$s invited you to conversation "%2$s".', [$user->getDisplayName(), $room->getDisplayName('')]);
 			$message->setFrom([Util::getDefaultEmailAddress('no-reply') => $user->getDisplayName()]);
 		} else {
-			$subject = $this->l->t('You were invited to a conversation.');
+			$subject = $this->l->t('You were invited to conversation "%s."', $room->getDisplayName(''));
 			$message->setFrom([Util::getDefaultEmailAddress('no-reply') => $this->defaults->getName()]);
 		}
 
 		$template->setSubject($subject);
 		$template->addHeader();
-		$template->addHeading($this->l->t('Conversation invitation'));
+		$template->addHeading(htmlspecialchars($room->getDisplayName('')), $room->getDisplayName(''));
```

Comment:
This feels weird in plain text emails, maybe we keep the old heading there:
```suggestion
		$template->addHeading(htmlspecialchars($room->getDisplayName('')), $this->l->t('Conversation invitation'));
```

![grafik](https://github.com/user-attachments/assets/57f3fb00-68b1-4339-a356-b52534cedbfb)

Reply:
Added "Click the link below" as a test instead for plaintext emails, and added an empty line.